x-pack/test/security_solution_api_integration/package.json

```diff
@@ -59,6 +59,11 @@
     "alerts:qa:serverless": "npm run run-tests:dr:default alerts serverless qaEnv",
     "alerts:server:ess": "npm run initialize-server:dr:default alerts ess",
     "alerts:runner:ess": "npm run run-tests:dr:default alerts ess essEnv",
+    "rule_management:server:serverless": "npm run initialize-server:dr:basicEssentials rule_management serverless",
+    "rule_management:runner:serverless": "npm run run-tests:dr:basicEssentials rule_management serverless serverlessEnv",
+    "rule_management:qa:serverless": "npm run run-tests:dr:basicEssentials rule_management serverless qaEnv",
+    "rule_management:server:ess": "npm run initialize-server:dr:basicEssentials rule_management ess",
+    "rule_management:runner:ess": "npm run run-tests:dr:basicEssentials rule_management ess essEnv",
```

Comment:
This are duplicated from line 141 onwards!

I think those need to be deleted.

Reply:
Good catch, left over a merge conflict 👍